proposals/incubating/COMMUNITY_FUND_PROCESS/COMMUNITY_FUND_PROCESS.md

```diff
@@ -0,0 +1,40 @@
+# Community fund request, approval, and reimbursement
+
+## Infrastructure
+
+* CPC voting members create budget committee
+  * Smaller number of seats to guarantee quorum
+  * Odd number of seats to avoid deadlocks
+  * Meets monthly
+  * Elected annually
+* New repo specific to the community fund (openjs-foundation/community-fund)
+  * Top-level README describing processes
+  * Budget file for each year containing the list of approved applicants, to be managed by the committee.
+* Mailing list for internal discussions between meetings
+* Issue template for applications, to be determined by budget committee
+* Labels for application stages, can only be changed by budget committee
+  * Under consideration
+  * Need more information
+  * Declined
+  * Accepted
+* LFX Crowdfunding used to disburse the payments, once approved.
+
+## Eligibility requirements
+
+* Must not have a source of corporate funding for the activity.
```

Comment:
Do we need to agree/get buy in from the board in terms of what kind of activities are reasonable to fund.  As written this lets the budget committee approve any kind of activity.

I'd been thinking (not necessarily the right way of thinking) that we'd propose in advance categories for example:
* certification scholarships
* training support, 
* etc.
And then have a process to approve these kinds of requests...

Reply:
Incidentals might be nice, anecdotally having just experienced technical problems, providing a means to replace damaged 2FA devices from community fund to be able to sign in to GH to continue developing Node would be great. Expanding upon this a little more, it would be nice if the project could expense standard-issue 2FA devices for all core collaborators.